src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -133,6 +133,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1015 = 1015,
 
+        /// <summary>
+        /// Project.Json project type is not supported.
```

Comment:
```suggestion
        /// project.Json project type is not supported.
```

Reply:
If we lowercase the project, we probably should also lowercase the json?

```suggestion
        /// project.json project type is not supported.
```